test-framework/src/runner.rs

```diff
@@ -236,3 +237,13 @@ pub fn run_tests<T: Runner>(runner: &T, expectation_category: &str) {
 
     std::env::remove_var("LEO_TESTFRAMEWORK");
 }
+
+/// Cleans the outputs by removing local paths.
+/// Needed for failing import tests as they require correct setting of CWD
+/// which leads to full system paths being printed out to the test expectations.
+pub fn sanitize_output(out: &String) -> String {
+    let mut leo_dir = std::env::current_dir().unwrap();
+    leo_dir.pop();
+    let leo_dir = leo_dir.into_os_string().into_string().unwrap();
```

Comment:
Consider making this computation a lazy static so that once it has been computed once, you need not do it again.